spec/loophp/collection/CollectionSpec.php

```diff
@@ -2499,6 +2503,20 @@ public function it_can_random(): void
             ->during('all');
     }
 
+    public function it_can_reduce(): void
```

Comment:
would be good to have another test case where the collection is a map, e.g. `Collection::fromIterable(['foo' => 'f', 'bar' => 'b'])`

Reply:
Done.